lib/ex_aws/ses.ex

```diff
@@ -0,0 +1,187 @@
+
+defmodule ExAws.SES do
+  import ExAws.Utils, only: [camelize_key: 1, camelize_keys: 1]
+
+  @moduledoc """
+  Operations on AWS SES
+
+  http://docs.aws.amazon.com/ses/latest/APIReference/Welcome.html
+  """
+
+  @doc "Verifies an email address"
+  @spec verify_email_identity(email :: binary) :: ExAws.Operation.Query.t
+  def verify_email_identity(email) do
+    request(:verify_email_identity, %{"EmailAddress" => email})
+  end
+
+  @doc "Fetch identities verification status and token (for domains)"
+  @spec get_identity_verification_attributes([binary]) :: ExAws.Operation.Query.t
+  def get_identity_verification_attributes(identities) when is_list(identities) do
```

Comment:
@lessless note that this went from `identity_verification_attributes` to `get_identity_verification_attributes` to more closely match AWS naming